bokeh/io.py

```diff
@@ -658,3 +664,75 @@ def export(obj, filename=None):
     image.save(filename)
 
     return os.path.abspath(filename)
+
+def _get_svgs(obj):
+    webdriver = import_required('selenium.webdriver',
+                                'To use bokeh.io.export you need selenium ' +
+                                '("conda install -c bokeh selenium" or "pip install selenium")')
+    # assert that phantomjs is in path for webdriver
+    detect_phantomjs()
+
+    html_path = tempfile.NamedTemporaryFile(suffix=".html").name
+    save(obj, filename=html_path, resources=INLINE, title="")
+
+    driver = webdriver.PhantomJS()
+    driver.get("file:///" + html_path)
+
+    _wait_until_render_complete(driver)
+
+    svg_script = """
+    var serialized_svgs = [];
+    var svgs = document.getElementsByClassName('bk-root')[0].getElementsByTagName("svg");
```

Comment:
there's never going to be a case where a SVG is under the ``bk-root`` node, is there?